### DISCUSSION TITLE
Javascript conversion error

### DISCUSSION THREAD

**Discussion by topologycapital** [pyodide/pyodide#2047] [Q&A] [answered]
Hi, I made a JSON file in Python and I converted it using tojs in Javascript and I keep on getting a python conversion error. Can anybody please help me?
```py
Piedata2=data[['Security','value']].rename(columns={"Security": "labels", "value": "values"})
pie_data1 =json.dumps(Piedata2.reset_index(drop=True).to_dict(orient='list'))
pie_data1=json.loads(pie_data1)
to_js(pie_data1, dict_converter=Object.fromEntries)

const pie_data = Object.fromEntries(pyodide.globals.toJs().get("pie_data1"));
```

**Comment by hoodmane** [comment, marked as answer]
Please include a complete example. Running the above code results in:
```py
NameError: name 'data' is not defined
```

**Reply by topologycapital** [reply]
I posted the file below, can you please help me to figure out what is going on?

**Comment by topologycapital** [comment]
[template copy 3.txt](https://github.com/pyodide/pyodide/files/7738864/template.copy.3.txt)

I continue to run into issues as listed below:
<img width="482" alt="Screen Shot 2021-12-17 at 10 29 32 PM" src="https://user-images.githubusercontent.com/85715782/146631844-ca3eb955-73cf-43ca-982b-b5c97ebfbcb9.png">

**Reply by hoodmane** [reply]
> `pyodide.globals.toJs()`

Don't do that. Maybe `pyodide.globals.get("pie_data1").toJs()` would work better.

**Comment by topologycapital** [comment]
Thank you very much for your help hoodmane
